src/framework/components/model/component.js

```diff
@@ -551,12 +557,11 @@ Object.assign(pc, function () {
 
                 var model = new pc.Model();
                 model.graph = node;
-
                 model.meshInstances = [new pc.MeshInstance(node, mesh, this._material)];
 
                 if (system._inTools)
                     model.generateWireframe();
-
+				this._clonedModel = true;
```

Comment:
Please change tabs for spaces (1 tab === 4 spaces)

Reply:
Why are you setting this? It's not a cloned model?

Reply:
If _clonemodel is not true, graphNode will have memory leaks, the model will not call destroy, and the node in meshinstance will not be assigned null.